commands.js

```diff
@@ -1782,8 +1782,18 @@ var commands = exports.commands = {
 
 	addplayer: function (target, room, user) {
 		if (!target) return this.parse('/help addplayer');
-
-		return this.parse('/roomplayer ' + target);
+		target = this.splitTarget(target, true);
+		var userid = toId(this.targetUsername);
+		var targetUser = this.targetUser;
+		var name = targetUser ? targetUser.name : this.targetUsername;
```

Comment:
Um, this does exactly the same thing as `var name = this.targetUsername`.

In other words, `this.targetUsername` already uses the target user's real name if it matches an online user.